components/atoms/ActionButton.js

```diff
@@ -56,6 +56,7 @@ export function ActionButton(props) {
         role="button"
         draggable="false"
         lang={props.lang}
+        locale={props.langUrl}
```

Comment:
What is the purpose of `props.langUrl`?

Reply:
It definitely doesn't do anything, It's not used in any of the pages, I'm removing it.